src/renderers/shaders/ShaderChunk/skinning_vertex.glsl

```diff
@@ -7,6 +7,7 @@
 	skinned += boneMatY * skinVertex * skinWeight.y;
 	skinned += boneMatZ * skinVertex * skinWeight.z;
 	skinned += boneMatW * skinVertex * skinWeight.w;
-	skinned  = bindMatrixInverse * skinned;
+
+	transformed = ( bindMatrixInverse * skinned ).xyz;
```

Comment:
You are not mutating `transformed`; you are overwriting it. So there are some implicit assumptions you are making. Can you please specify what they are?

Are you able to demonstrate that the w-component you are dropping is 1?